### PR TITLE
TRT-1803: Wait for operators to become stable before test

### PR DESCRIPTION
**Pull request by xueqzhan**

### CODE REVIEW

File: pkg/clioptions/clusterinfo/wait_for_stable_cluster.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+		}, err
+	}
+
+	interval := 10 * time.Second
+	timeout := 10 * time.Minute
+	minimumStablePeriod := 3 * time.Minute
+	recoveredOperators := map[string]string{}
+	unstableOperators := map[string]string{}
+	var stabilityStarted *time.Time
+	waitErr := wait.PollUntilContextTimeout(ctx, interval, timeout, true, func(waitCtx context.Context) (bool, error) {
```
Comment (pacevedom) [inline pkg/clioptions/clusterinfo/wait_for_stable_cluster.go:66]: You may want to skip this one for MicroShift, as it does not have config apigroup.

Comment (xueqzhan) [reply pkg/clioptions/clusterinfo/wait_for_stable_cluster.go:100]: Good catch! Thanks

**Pull request merged**